tasks/build/docker/docs/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+services:
+  kibana_docs:
+    container_name: $KIBANA_DOCS_CONTAINER_NAME
```

Comment:
This could be hardcoded and the ENV variable removed.

Reply:
i forgot, this is used to get the container id so we can copy the docs out